### ISSUE TITLE
FADBlock no longer undetected

### ISSUE THREAD

**Issue by Lars4k** [0x48piraj/fadblock#88]
Since today youtube recognizes FADBlock and warns the user. FADBlock is no longer undetected. It no longer works.

**Comment by harkai84** [comment]
I can not confirm.
Can you take a screenshot?
What browser are you using and what version of Fadblock do you have?

**Comment by lesoudeur** [comment]
I am using Chrome ver 119.0.6045.124 (Official Build) (64-bit). FadBlock Version 2.1. 
![image](https://github.com/0x48piraj/fadblock/assets/46300085/b84ec2b7-68ad-41ab-bbbe-b9044f64ac44)

**Comment by harkai84** [comment]
Do you have YouTube whitelisted on your other ad blocker?
Which extensions do you have installed?
You can also take a screenshot of your extensions if you are unsure.
You should disable or uninstall all ad blockers except Fadblock for Youtube.
I looked again, but I don't get this message.

**Comment by lesoudeur** [comment]
Have disabled all other ad blockers for You Tube. Installed extensions
![image](https://github.com/0x48piraj/fadblock/assets/46300085/d2d3f6dd-0849-4c71-b4dc-733d63c9bac3)

**Comment by harkai84** [comment]
Nope. Adblock Plus - free ad blocker and uBlock Origin are installed.
Try to disable them and try it again.
If its work, you can put Youtube.com on a whitelist in uBlock Origin and disable Adblock Plus - free ad blocker.

**Comment by lesoudeur** [comment]
Nope. Adblock Plus - free ad blocker and uBlock Origin are installed.

Try to disable them and try it again.

Both of these are blocked for You tube though they are active for other web sites.

I have also tried disabling Adblock Plus - free ad blocker

 

Am still getting blocked in You Tube

**Comment by harkai84** [comment]
In Extensions you can deactivate any extension by pressing the switch at the bottom right once.
Then it turns gray.
This is what you do with Adblock Plus & uBlock Origin.
If you have problems with your browser, try another browser with Fadblock.
Maybe Firefox.
I can only repeat myself, but Fadblock works perfectly for me.

**Comment by kiki67100** [comment]
Try to switch Google account or disconnect. 
I had blocked from Youtube with my account (all videos requests in network was 403)

**Comment by lesoudeur** [comment]
It is working in Edge ATM.

**Comment by harkai84** [comment]
That looks good.
Now you just need to figure out why it isn't working on your Chrome browser.
But since it doesn't seem to be due to Fadblock, could this topic be closed or do you have another question?

**Comment by lesoudeur** [comment]
No, if I find out why it is not working in Chrome (especially when Edge is now Chrome based) and it is in Edge with the same extensions and the same extension settings _except Fad block which is off_. So I am not getting blocked in You Tube ATM without using Fad Block. I will report back when I get more stability.

**Comment by 0x48piraj** [comment]
Thank you a lot @harkai84 for always helping out! You're a gem!

@lesoudeur. similar threads have been opened with vague details, so it's okay to post your report there, too many open threads cause people to not find the solution in one place. I will be leaving this open for now. If you find anything, please post it here. If there's a geo-specific patch, I would like to quickly squash the report before it's deployed globally.

**Comment by Jacksonant** [comment]
@0x48piraj  fadblock is not working anymore at my side. Just not working, but without any warning/block from youtube.

**Comment by TrancEditi0n** [comment]
@0x48piraj Same here. Was working yesterday but not now, the same thing happens to me as Jacksonant

**Comment by 0x48piraj** [comment]
I am looking into this. Can you guys post your DOM state so I can analyze more reports?

**Comment by TrancEditi0n** [comment]
> I am looking into this. Can you guys post your DOM state so I can analyze more reports?
> 
> Do you have uBlock Origin installed and enabled as the OP had?
> 
> Because if that's the case, your account has probably been detected using uBlock and has been banned. Here's the report: #87

@0x48piraj 

[DOM Ad Pause.txt](https://github.com/0x48piraj/fadblock/files/13315342/DOM.Ad.Pause.txt)

Here is my DOM state at the time I have paused an ad.

My extensions:

![image](https://github.com/0x48piraj/fadblock/assets/112038736/dbb3dcde-9397-4c2a-b705-b38c1809a8a9)

**Comment by 0x48piraj** [comment]
I will look into the DOM but it works for people who don't have any traditional adblockers. I see you using AdBlock Plus/Ultimate (it's not enabled but if it was at any duration, that can cause a problem).

**Comment by TrancEditi0n** [comment]
I uninstalled AdBlock completely, restarted chrome but it didn't solve it.

Thanks for the effort!

**Comment by 0x48piraj** [comment]
Once your account is blocked or the device is flagged, YouTube stores that ban data so uninstalling after won't do much.

If possible, hop on a new device with a different network, with a different YouTube account. Install FadBlock v2.1 on Chrome (you can login with the same email if you have a lifetime license) and test it again.

As it seems it's working for people who are just using FadBlock and nothing else.

**Comment by 0x48piraj** [comment]
Kindly refrain from using alternative ad blockers, especially if you've supported FadBlock through a donation. Some users combining multiple ad blockers, against the FAQ's advice, are facing detection issues, resulting in YouTube account bans. Iit's not a FadBlock issue in any way. It is working for people who are only using FadBlock.

**Note:** If YouTube has banned you, removing Adblocker Plus/Ultimate, uBlock won't work as YouTube has logged your IP, device and account data into their ban list.

Here's a Reddit thread which proves the above remark: https://www.reddit.com/r/Adblock/comments/17rluvd/purging_caches_refreshing_in_ublock_on_firefox/

**Comment by Jacksonant** [comment]
This is the dom when i pause video with ads running. I have no other adblock, in fact i never install any adblock extension in my browser (chrome).
[adblock_stop_dom.txt](https://github.com/0x48piraj/fadblock/files/13319590/adblock_stop_dom.txt)

**Comment by TrancEditi0n** [comment]
> Kindly refrain from using alternative ad blockers, especially if you've supported FadBlock through a donation. Some users combining multiple ad blockers, against the FAQ's advice, are facing detection issues, resulting in YouTube account bans. It's not a FadBlock issue in any way. It is working for people who are only using FadBlock.
> 
> **Note:** If YouTube has banned you, removing Adblocker Plus/Ultimate, uBlock won't work as YouTube has logged your IP, device and account data into their ban list.
> 
> Here's a Reddit thread that proves the above remark: https://www.reddit.com/r/Adblock/comments/17rluvd/purging_caches_refreshing_in_ublock_on_firefox/

Unfortunately I can't try with another PC so I'm currently trying with ublock (cleaning filters and updating) and for now it seems to work correctly.

As soon as FADBlock works again, I'm going to use it again
